mappings/net/minecraft/server/world/ServerWorld.mapping

```diff
@@ -193,14 +253,38 @@ CLASS net/minecraft/class_3218 net/minecraft/server/world/ServerWorld
 	METHOD method_27056 getStructureAccessor ()Lnet/minecraft/class_5138;
 	METHOD method_27909 iterateEntities ()Ljava/lang/Iterable;
 	METHOD method_27910 setWeather (IIZZ)V
+		COMMENT Sets the current weather, as well as how long it should last.
+		COMMENT
+		COMMENT @see ServerWorldProperties#setClearWeatherTime
+		COMMENT @see ServerWorldProperties#setRainTime
+		COMMENT @see ServerWorldProperties#setThunderTime
+		COMMENT @see ServerWorldProperties#setRaining
+		COMMENT @see ServerWorldProperties#setThundering
 		ARG 1 clearDuration
+			COMMENT how long the clear weather should last, in seconds
 		ARG 2 rainDuration
+			COMMENT how long the rain or the thunderstorm should last, in seconds
 		ARG 3 raining
+			COMMENT whether a rain is ongoing
 		ARG 4 thundering
+			COMMENT whether a thunderstorm is ongoing
 	METHOD method_27911 getSpawnPos ()Lnet/minecraft/class_2338;
+		COMMENT {@return the world spawn point}
+		COMMENT
+		COMMENT @implNote If it is outside the world border, this returns the position of the
+		COMMENT highest block at the center of the world border.
```

Comment:
Highest block or highest non-air block?

Reply:
also minecraft worlds have heightmaps too

Reply:
Highest motion-blocking block. I should add that.